src/libraries/System.Net.Http/ref/System.Net.Http.cs

```diff
@@ -97,17 +97,17 @@ public partial class HttpClientHandler : System.Net.Http.HttpMessageHandler
         public System.Net.Http.ClientCertificateOption ClientCertificateOptions { get { throw null; } set { } }
         public System.Security.Cryptography.X509Certificates.X509CertificateCollection ClientCertificates { get { throw null; } }
         public System.Net.CookieContainer CookieContainer { get { throw null; } set { } }
-        public System.Net.ICredentials Credentials { get { throw null; } set { } }
+        public System.Net.ICredentials? Credentials { get { throw null; } set { } }
         public static System.Func<System.Net.Http.HttpRequestMessage, System.Security.Cryptography.X509Certificates.X509Certificate2, System.Security.Cryptography.X509Certificates.X509Chain, System.Net.Security.SslPolicyErrors, bool> DangerousAcceptAnyServerCertificateValidator { get { throw null; } }
-        public System.Net.ICredentials DefaultProxyCredentials { get { throw null; } set { } }
+        public System.Net.ICredentials? DefaultProxyCredentials { get { throw null; } set { } }
         public int MaxAutomaticRedirections { get { throw null; } set { } }
         public int MaxConnectionsPerServer { get { throw null; } set { } }
         public long MaxRequestContentBufferSize { get { throw null; } set { } }
         public int MaxResponseHeadersLength { get { throw null; } set { } }
         public bool PreAuthenticate { get { throw null; } set { } }
         public System.Collections.Generic.IDictionary<string, object> Properties { get { throw null; } }
```

Comment:
Should this be `IDictionary<string, object?>` to allow null values?

Reply:
I don't see any code accessing dictionary elements, so I guess I can make value nullable